manifests/w/WaveMetrics/IgorPro9/9.05/WaveMetrics.IgorPro9.yaml

```
@@ -0,0 +1,8 @@
+# Created with YamlCreate.ps1 v2.3.2 $debug=NVS1.CRLF.7-4-1.Win32NT
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.version.1.5.0.schema.json
+
+PackageIdentifier: WaveMetrics.IgorPro9
+PackageVersion: "9.05"
```

```suggestion
PackageVersion: 9.0.5.1
```
This should match control panel. After changing this, you will also need to rename the folder.